typespec-extension/src/code-model-builder.ts

```diff
@@ -387,7 +391,10 @@ export class CodeModelBuilder {
         const operations = listOperationsInOperationGroup(this.sdkContext, operationGroup);
         codeModelGroup = new OperationGroup(operationGroup.type.name);
         for (const operation of operations) {
-          codeModelGroup.addOperation(this.processOperation(operationGroup.type.name, operation, clientContext));
+          const codeModelOperation = this.processOperation(operationGroup.type.name, operation, clientContext);
```

Comment:
Why not filter by `getOverloadedOperation` here?

If one can avoid `undefined` in type, avoid it.

Reply:
good suggestion, will update. Thanks